Detectors/TOF/compression/src/CompressorTask.cxx

```diff
@@ -150,6 +150,9 @@ void CompressorTask<RDH, verbose, paranoid>::run(ProcessingContext& pc)
       headerOut.payloadSize += payloadOutSize;
     }
 
+    if (headerOut.payloadSize < 0) {
```

Comment:
`DataHeader::payloadSize` is `uint64_t`, this will be always evaluated to false, even in case of the overflow in the additions above.

Reply:
You are right, then I have to deeper investigate

Reply:
I removed this line in the meanwhile